apps/jobs/urls.py

```diff
@@ -3,16 +3,16 @@
 from . import views
 
 urlpatterns = [
-    url(r'challenge/(?P<challenge_pk>[0-9]+)/'
+    url(r'^challenge/(?P<challenge_pk>[0-9]+)/$'
         r'challenge_phase/(?P<challenge_phase_pk>[0-9]+)/submission/(?P<submission_pk>[0-9]+)',
         views.change_submission_data_and_visibility, name='change_submission_data_and_visibility'),
-    url(r'challenge/(?P<challenge_id>[0-9]+)/'
+    url(r'^challenge/(?P<challenge_id>[0-9]+)/$'
         r'challenge_phase/(?P<challenge_phase_id>[0-9]+)/submission/',
         views.challenge_submission, name='challenge_submission'),
-    url(r'(?P<challenge_pk>[0-9]+)/'
+    url(r'^(?P<challenge_pk>[0-9]+)/$'
```

Comment:
Same as above.

Reply:
@RishabhJain2018 I didn't saw it, sorry my mistake.